### PR TITLE
chore(Renovate): Wait for a release to reach a minimum age

### PR DESCRIPTION
**Pull request by kvanzuijlen**
### What does this PR do?
Added a configuration to wait 12 hours before creating a PR for new releases

- Fixes #

If you modified files in the `./charts/jenkins/` directory, please also include the following:

```[tasklist]
### Submitter checklist
- [ ] I bumped the "version" key in `./charts/jenkins/Chart.yaml`.
- [ ] I added a new changelog entry to `./charts/jenkins/CHANGELOG.md`.
- [x] I followed the [technical requirements](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#technical-requirements).
```

### Special notes for your reviewer
I also configured the hourly limit, increasing it to 5 from the normal 2.

### CODE REVIEW

**Comment by NotMyFault** [comment]
The change proposed should ignore lts line updates.

**Comment by NotMyFault** [comment]
> > The change proposed should ignore lts line updates.
> 
> Do you mean that jenkins lts versions shouldn't wait for 12 hours? That's possible. Let me know.

Yes.

**Comment by kvanzuijlen** [comment]
This should exclude the minimumReleaseAge check for jenkins lts

File: .github/renovate-config.json5 [hunk 1]
```diff
@@ -3,10 +3,11 @@
   onboarding: false,
   requireConfig: "ignored",
   platform: "github",
-  repositories: ["jenkinsci/helm-charts"],
+//  repositories: ["jenkinsci/helm-charts"],
```
Comment (timja) [inline .github/renovate-config.json5:6]:
```suggestion
  repositories: ["jenkinsci/helm-charts"],
```

Comment (kvanzuijlen) [reply .github/renovate-config.json5:6]: Whoops, yeah, thanks for catching this. Had to comment that to run Renovate locally.

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**